src/promptflow/promptflow/_sdk/operations/_flow_operations.py

```diff
@@ -531,7 +531,7 @@ def build(
         :return: no return
         :rtype: None
         """
-        output_dir = Path(output)
+        output_dir = Path(output).absolute()
         output_dir.mkdir(parents=True, exist_ok=True)
```

Comment:
Please help reply the conversation once PR is done: https://github.com/microsoft/promptflow/discussions/1190